packages/badge/test/badge.test.ts

```diff
@@ -46,5 +48,40 @@ describe('Badge', () => {
         await elementUpdated(el);
 
         await expect(el).to.be.accessible();
+        expect(consoleWarnStub.called).to.be.false;
+        consoleWarnStub.restore();
+    });
+    it('warns in Dev Mode when sent an incorrect `variant`', async () => {
```

Comment:
We could add a TODO to build a utility function to make setting up these dev mode tests more standardized for future components.